docs/install/from_source.rst

```diff
@@ -73,6 +74,15 @@ linux operating systems, execute (in a terminal):
     sudo apt-get update
     sudo apt-get install -y python3 python3-dev python3-setuptools gcc libtinfo-dev zlib1g-dev build-essential cmake libedit-dev libxml2-dev
 
+On macOS, whether using an Intel or M1 Mac, we support using homebrew. Make sure to follow the post installation steps for each of the below commaands.
```

Comment:
sp: commaands

Suggested edit:

"Use Homebrew to install the required dependencies for macOS running either the Intel or M1 processors. You must follow the post-installation steps specified by Homebrew to ensure the dependencies are correctly installed and configured."

Reply:
Done